chromium_src/third_party/blink/renderer/modules/plugins/dom_plugin_array.cc

```diff
@@ -136,11 +136,10 @@ void FarblePlugins(DOMPluginArray* owner,
       dom_plugins->push_back(fake_dom_plugin_2);
       // Shuffle the list of plugins pseudo-randomly, based on the domain key.
       std::shuffle(dom_plugins->begin(), dom_plugins->end(), prng);
-      break;
+      return;
     }
-    default:
-      NOTREACHED_IN_MIGRATION();
   }
+  NOTREACHED();
```

Comment:
nit: `NOTREACHED() << "Unexpected value for farbling_level: "<< base::to_underlying(farbling_level);`